varcode/variant.py

```diff
@@ -247,83 +236,54 @@ def __eq__(self, other):
             self.alt == other.alt and
             self.ensembl == other.ensembl)
 
-    def __getstate__(self):
-        # When pickle.load'ing, we want the original values vs. the normalized values.
-        # Normalization will happen upon object creation.
-        return [
-            self.contig,
-            self.original_start,
-            self.original_ref,
-            self.original_alt,
-            self.ensembl.release
-        ]
-
-    def __setstate__(self, fields):
-        # Special logic needed for JSON serialization, since EnsemblRelease
-        # is not JSON serializable.
-        # TODO: this does not work for Genome objects; only EnsemblRelease.
-        ensembl = cached_release(fields.pop())
-        fields.append(ensembl)
-
-        self.__init__(*fields)
-
-    def to_json(self):
+    def to_dict(self):
         """
-        Serialize to JSON.
-
-        Returns
-        ----------
-        String giving the Variant instance serialized in JSON format.
-
-        """
-        return json.dumps(self.__getstate__())
-
-    @classmethod
-    def from_dict(cls, state):
+        We want the original values (un-normalized) field values while
+        serializing since normalization will happen in __init__.
         """
-        Unserialize a Variant encoded as a Python dict.
-        """
-        instance = cls.__new__(cls)
-        instance.__setstate__(state)
-        return instance
-
-    @classmethod
-    def from_json(cls, serialized):
-        """
-        Unserialize a Variant encoded as a string of JSON.
-        """
-        return Variant.from_dict(json.loads(serialized))
+        return dict(
+            contig=self.contig,
```

Comment:
Also a good catch. Excited to eventually demote "normalization" to properties and have start/ref/alt be the same values as in a VCF.